locust/main.py

```diff
@@ -100,6 +100,24 @@ def main():
             user_classes[key] = value
             available_user_classes[key] = value
 
+    if len(stats.PERCENTILES_TO_CHART) > 2:
+        sys.stderr.write("stats.PERCENTILES_TO_CHART parameter can be specified up tp 2 parameters \n")
```

Comment:
Use logging.error instead

Reply:
Sure, I'll do like this

    if len(stats.PERCENTILES_TO_CHART) > 2:
        logging.error("stats.PERCENTILES_TO_CHART parameter can be specified up tp 2 parameters \n")
        sys.exit(1)

Reply:
👍